docs/architecture/psa-migration/tasks-g1.md

```diff
@@ -0,0 +1,114 @@
+This document is temporary; it lists tasks to achieve G1 as described in
```

Comment:
Given that we have github issues for everything - is this document now obsolete? (No objection to retaining it for reference, but should update the text here).